starters/gatsby-wordpress-starter/lib/PaginationClient.js

```diff
@@ -0,0 +1,5 @@
+import { GraphqlClientFactory } from '@pantheon-systems/wordpress-kit'
+
+export const paginationClient = new GraphqlClientFactory(
+	'https://dev-decoupled-wp-mock-data.pantheonsite.io/wp/graphql',
+).create()
```

Comment:
I think this client can move to the `PostsPagination.js` file since we aren't reusing it anywhere else.